src/api/handlers/member.ts

```diff
@@ -240,6 +240,11 @@ export function moveMember(
   return editMember(guildID, memberID, { channel_id: channelID });
 }
 
+/** Kicks a member from a voice channel */
+export function kickFromVoiceChannel(guildID: string, memberID: string) {
```

Comment:
Is it possible to add getters for this on some of the structs? For example, `channel.kickFromVoice()`, `member.kickFromVoice()`, `guild.kickFromVoice()`

Reply:
MilSci has already added getters. He is waiting for this PR to be merged.